<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>762</width>
    <height>772</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>我的WPS</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QComboBox" name="comboBox">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>181</width>
      <height>22</height>
     </rect>
    </property>
    <item>
     <property name="text">
      <string>标准</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>项目符号(●)</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>项目符号(○)</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>项目符号(■)</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>项目符号(1.2.3.)</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>项目符号(a.b.c.)</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>项目符号(A.B.C.)</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>项目符号(i.ii.iii.)</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>项目符号(I.II.III.)</string>
     </property>
    </item>
   </widget>
   <widget class="QFontComboBox" name="fontComboBox">
    <property name="geometry">
     <rect>
      <x>180</x>
      <y>0</y>
      <width>221</width>
      <height>22</height>
     </rect>
    </property>
   </widget>
   <widget class="QComboBox" name="sizeComboBox">
    <property name="geometry">
     <rect>
      <x>400</x>
      <y>0</y>
      <width>87</width>
      <height>22</height>
     </rect>
    </property>
   </widget>
   <widget class="QMdiArea" name="mdiArea">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>20</y>
      <width>761</width>
      <height>661</height>
     </rect>
    </property>
    <property name="toolTipDuration">
     <number>-5</number>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>762</width>
     <height>21</height>
    </rect>
   </property>
   <widget class="QMenu" name="menu_F">
    <property name="title">
     <string>文件(&amp;F)</string>
    </property>
    <addaction name="newAction"/>
    <addaction name="openAction"/>
    <addaction name="saveAction"/>
    <addaction name="saveAsAction"/>
    <addaction name="printAction"/>
    <addaction name="printPreviewAction"/>
    <addaction name="exitAction"/>
   </widget>
   <widget class="QMenu" name="menu_E">
    <property name="title">
     <string>编辑(&amp;E)</string>
    </property>
    <addaction name="undoAction"/>
    <addaction name="redoAction"/>
    <addaction name="cutAction"/>
    <addaction name="copyAction"/>
    <addaction name="pasteAction"/>
   </widget>
   <widget class="QMenu" name="menu_O">
    <property name="title">
     <string>格式(&amp;O)</string>
    </property>
    <widget class="QMenu" name="menu_D">
     <property name="title">
      <string>字体(D)</string>
     </property>
     <addaction name="boldAction"/>
     <addaction name="italicAction"/>
     <addaction name="underlineAction"/>
    </widget>
    <widget class="QMenu" name="menu">
     <property name="title">
      <string>段落</string>
     </property>
     <addaction name="leftAlignAction"/>
     <addaction name="centerAction"/>
     <addaction name="rightAlignAction"/>
     <addaction name="justifyAction"/>
    </widget>
    <addaction name="menu_D"/>
    <addaction name="menu"/>
    <addaction name="colorAction"/>
   </widget>
   <widget class="QMenu" name="menu_W">
    <property name="title">
     <string>窗体(&amp;W)</string>
    </property>
    <addaction name="closeAction"/>
    <addaction name="closeAllAction"/>
    <addaction name="titleAction"/>
    <addaction name="cascadeAction"/>
    <addaction name="nextAction"/>
    <addaction name="previousAction"/>
   </widget>
   <widget class="QMenu" name="menu_H">
    <property name="title">
     <string>帮助(&amp;H)</string>
    </property>
    <addaction name="aboutAction"/>
   </widget>
   <addaction name="menu_F"/>
   <addaction name="menu_E"/>
   <addaction name="menu_O"/>
   <addaction name="menu_W"/>
   <addaction name="menu_H"/>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
   <addaction name="newAction"/>
   <addaction name="openAction"/>
   <addaction name="saveAction"/>
   <addaction name="printAction"/>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
  <widget class="QToolBar" name="toolBar">
   <property name="windowTitle">
    <string>toolBar</string>
   </property>
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
   <addaction name="undoAction"/>
   <addaction name="redoAction"/>
   <addaction name="cutAction"/>
   <addaction name="copyAction"/>
   <addaction name="pasteAction"/>
  </widget>
  <widget class="QToolBar" name="toolBar_2">
   <property name="windowTitle">
    <string>toolBar_2</string>
   </property>
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
   <addaction name="colorAction"/>
   <addaction name="boldAction"/>
   <addaction name="italicAction"/>
   <addaction name="underlineAction"/>
   <addaction name="separator"/>
   <addaction name="leftAlignAction"/>
   <addaction name="centerAction"/>
   <addaction name="rightAlignAction"/>
   <addaction name="justifyAction"/>
  </widget>
  <action name="newAction">
   <property name="icon">
    <iconset resource="image.qrc">
     <normaloff>:/images/new.png</normaloff>:/images/new.png</iconset>
   </property>
   <property name="text">
    <string>新建(N)</string>
   </property>
   <property name="statusTip">
    <string>创建一个新文档</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+N</string>
   </property>
  </action>
  <action name="openAction">
   <property name="icon">
    <iconset resource="image.qrc">
     <normaloff>:/images/fileopen.png</normaloff>:/images/fileopen.png</iconset>
   </property>
   <property name="text">
    <string>打开(O）</string>
   </property>
   <property name="statusTip">
    <string>打开一个已存在的文档</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+O</string>
   </property>
  </action>
  <action name="saveAction">
   <property name="icon">
    <iconset resource="image.qrc">
     <normaloff>:/images/save.png</normaloff>:/images/save.png</iconset>
   </property>
   <property name="text">
    <string>保存(S)</string>
   </property>
   <property name="statusTip">
    <string>将当前的文档存盘</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+S</string>
   </property>
  </action>
  <action name="saveAsAction">
   <property name="text">
    <string>另存为(A)...</string>
   </property>
   <property name="statusTip">
    <string>以一个新名字保存文档</string>
   </property>
  </action>
  <action name="printAction">
   <property name="icon">
    <iconset resource="image.qrc">
     <normaloff>:/images/print.png</normaloff>:/images/print.png</iconset>
   </property>
   <property name="text">
    <string>打印(P)...</string>
   </property>
   <property name="statusTip">
    <string>打印输出文档</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+P</string>
   </property>
  </action>
  <action name="printPreviewAction">
   <property name="text">
    <string>打印预览...</string>
   </property>
   <property name="statusTip">
    <string>预览打印效果</string>
   </property>
  </action>
  <action name="exitAction">
   <property name="text">
    <string>退出</string>
   </property>
   <property name="statusTip">
    <string>退出应用程序</string>
   </property>
  </action>
  <action name="undoAction">
   <property name="icon">
    <iconset resource="image.qrc">
     <normaloff>:/images/undo.png</normaloff>:/images/undo.png</iconset>
   </property>
   <property name="text">
    <string>撤销(U)</string>
   </property>
   <property name="statusTip">
    <string>撤销当前的操作</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+Z</string>
   </property>
  </action>
  <action name="redoAction">
   <property name="icon">
    <iconset resource="image.qrc">
     <normaloff>:/images/redo.png</normaloff>:/images/redo.png</iconset>
   </property>
   <property name="text">
    <string>重做(R)</string>
   </property>
   <property name="statusTip">
    <string>恢复之前的操作</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+Y</string>
   </property>
  </action>
  <action name="cutAction">
   <property name="icon">
    <iconset resource="image.qrc">
     <normaloff>:/images/cut.png</normaloff>:/images/cut.png</iconset>
   </property>
   <property name="text">
    <string>剪切(T)</string>
   </property>
   <property name="statusTip">
    <string>从文档中裁剪所选的内容，并放入剪切板</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+X</string>
   </property>
  </action>
  <action name="copyAction">
   <property name="icon">
    <iconset resource="image.qrc">
     <normaloff>:/images/copy.png</normaloff>:/images/copy.png</iconset>
   </property>
   <property name="text">
    <string>复制(C)</string>
   </property>
   <property name="statusTip">
    <string>复制所选的内容，并放入剪切板</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+C</string>
   </property>
  </action>
  <action name="pasteAction">
   <property name="icon">
    <iconset resource="image.qrc">
     <normaloff>:/images/paste.png</normaloff>:/images/paste.png</iconset>
   </property>
   <property name="text">
    <string>粘贴(P)</string>
   </property>
   <property name="statusTip">
    <string>将剪切板的内容粘贴到文档</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+V</string>
   </property>
  </action>
  <action name="colorAction">
   <property name="icon">
    <iconset resource="image.qrc">
     <normaloff>:/images/color.ico</normaloff>:/images/color.ico</iconset>
   </property>
   <property name="text">
    <string>颜色(C)...</string>
   </property>
   <property name="statusTip">
    <string>设置文字颜色</string>
   </property>
  </action>
  <action name="boldAction">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="icon">
    <iconset resource="image.qrc">
     <normaloff>:/images/bold.png</normaloff>:/images/bold.png</iconset>
   </property>
   <property name="text">
    <string>加粗(B)</string>
   </property>
   <property name="statusTip">
    <string>将所选文字加粗</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+B</string>
   </property>
  </action>
  <action name="italicAction">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="icon">
    <iconset resource="image.qrc">
     <normaloff>:/images/italic.png</normaloff>:/images/italic.png</iconset>
   </property>
   <property name="text">
    <string>倾斜(I)</string>
   </property>
   <property name="statusTip">
    <string>将所选文字用斜体显示</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+I</string>
   </property>
  </action>
  <action name="underlineAction">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="icon">
    <iconset resource="image.qrc">
     <normaloff>:/images/under.png</normaloff>:/images/under.png</iconset>
   </property>
   <property name="text">
    <string>下划线(U)</string>
   </property>
   <property name="statusTip">
    <string>将所选文字加下划线</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+U</string>
   </property>
  </action>
  <action name="leftAlignAction">
   <property name="icon">
    <iconset resource="image.qrc">
     <normaloff>:/images/left.png</normaloff>:/images/left.png</iconset>
   </property>
   <property name="text">
    <string>左对齐(L)</string>
   </property>
   <property name="statusTip">
    <string>将文字左对齐</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+L</string>
   </property>
  </action>
  <action name="centerAction">
   <property name="icon">
    <iconset resource="image.qrc">
     <normaloff>:/images/center.png</normaloff>:/images/center.png</iconset>
   </property>
   <property name="text">
    <string>居中(C)</string>
   </property>
   <property name="statusTip">
    <string>将所选文字居中</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+E</string>
   </property>
  </action>
  <action name="rightAlignAction">
   <property name="icon">
    <iconset resource="image.qrc">
     <normaloff>:/images/right.png</normaloff>:/images/right.png</iconset>
   </property>
   <property name="text">
    <string>右对齐(R)</string>
   </property>
   <property name="statusTip">
    <string>将所选文字右对齐</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+R</string>
   </property>
  </action>
  <action name="justifyAction">
   <property name="icon">
    <iconset resource="image.qrc">
     <normaloff>:/images/justify.png</normaloff>:/images/justify.png</iconset>
   </property>
   <property name="text">
    <string>两端对齐(J)</string>
   </property>
   <property name="statusTip">
    <string>将所选文字左右两端同时对齐，并根据需要调整间距</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+J</string>
   </property>
  </action>
  <action name="closeAction">
   <property name="text">
    <string>关闭(O)</string>
   </property>
   <property name="statusTip">
    <string>关闭活动子窗口</string>
   </property>
  </action>
  <action name="closeAllAction">
   <property name="text">
    <string>关闭所有(A)</string>
   </property>
   <property name="statusTip">
    <string>关闭所有子窗口</string>
   </property>
  </action>
  <action name="titleAction">
   <property name="text">
    <string>平铺(T)</string>
   </property>
   <property name="statusTip">
    <string>平铺子窗口</string>
   </property>
  </action>
  <action name="cascadeAction">
   <property name="text">
    <string>层叠(C)</string>
   </property>
   <property name="statusTip">
    <string>层叠子窗口</string>
   </property>
  </action>
  <action name="nextAction">
   <property name="text">
    <string>下一个(X)</string>
   </property>
   <property name="statusTip">
    <string>移动焦点到下一个子窗口</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+Tab</string>
   </property>
  </action>
  <action name="previousAction">
   <property name="text">
    <string>前一个(V)</string>
   </property>
   <property name="statusTip">
    <string>移动焦点到前一个子窗口</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+Shift+Tab</string>
   </property>
  </action>
  <action name="aboutAction">
   <property name="text">
    <string>关于(A)</string>
   </property>
   <property name="statusTip">
    <string>关于我的WPS v1.0的内容</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources>
  <include location="image.qrc"/>
 </resources>
 <connections/>
</ui>
